tests/components/deconz/test_sensor.py

```diff
@@ -146,6 +128,10 @@ async def test_sensors(hass):
     assert consumption_sensor.state == "0.002"
     assert "device_class" not in consumption_sensor.attributes
 
+    assert hass.states.get("sensor.clip_light_level_sensor") is None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)